docs/developers/contributing.md

```diff
@@ -72,6 +72,6 @@ Guidelines for reporting bugs:
 
  - Check the issue search to see if it has already been reported
  - Isolate the problem to a simple test case
- - Please include a demonstration of the bug on a website such as [JS Bin](https://jsbin.com/), [JS Fiddle](https://jsfiddle.net/), or [Codepen](https://codepen.io/pen/). ([Template](https://codepen.io/pen?template=JXVYzq))
+ - Please include a demonstration of the bug on a website such as [JS Bin](https://jsbin.com/), [JS Fiddle](https://jsfiddle.net/), or [Codepen](https://codepen.io/pen/). ([Template](https://codepen.io/pen?template=JXVYzq)). If filing a bug against `master`, you may reference the latest code via https://www.chartjs.org/dist/master/Chart.min.js (changing the filename to point at the file you need as appropriate).
```

Comment:
Should that template be updated to use master by default?

Reply:
I would instead update the template with the latest release: https://cdn.jsdelivr.net/npm/chart.js/dist/Chart.min.js

Reply:
The template is using https://npmcdn.com/chart.js@latest/dist/Chart.bundle.min.js, so it's already using the latest release, isn't it?